From 6177e74134042008d5a41ae3df9726988e96f886 Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Mon, 27 Aug 2018 21:23:47 +0200
Subject: [PATCH 053/293] xlat: provide fallback definitions for epollevents

* xlat/epollevents.in: Provide fallback definitions, add EPOLLNVAL
constant (introduced by Linux commit v4.16-rc1~3^2~3).
---
 xlat/epollevents.in | 31 ++++++++++++++++---------------
 1 file changed, 16 insertions(+), 15 deletions(-)

diff --git a/xlat/epollevents.in b/xlat/epollevents.in
index 6fa6707..2dced00 100644
--- a/xlat/epollevents.in
+++ b/xlat/epollevents.in
@@ -1,15 +1,16 @@
-EPOLLIN
-EPOLLPRI
-EPOLLOUT
-EPOLLRDNORM
-EPOLLRDBAND
-EPOLLWRNORM
-EPOLLWRBAND
-EPOLLMSG
-EPOLLERR
-EPOLLHUP
-EPOLLRDHUP
-EPOLLEXCLUSIVE
-EPOLLWAKEUP
-EPOLLONESHOT
-EPOLLET
+EPOLLIN		0x00000001
+EPOLLPRI	0x00000002
+EPOLLOUT	0x00000004
+EPOLLERR	0x00000008
+EPOLLHUP	0x00000010
+EPOLLNVAL	0x00000020
+EPOLLRDNORM	0x00000040
+EPOLLRDBAND	0x00000080
+EPOLLWRNORM	0x00000100
+EPOLLWRBAND	0x00000200
+EPOLLMSG	0x00000400
+EPOLLRDHUP	0x00002000
+EPOLLEXCLUSIVE	0x10000000
+EPOLLWAKEUP	0x20000000
+EPOLLONESHOT	0x40000000
+EPOLLET		0x80000000
-- 
1.7.12.4

